modules/toggle.js

```diff
@@ -0,0 +1,11 @@
+const toggleCheckbox = (todoArray) => {
+  const chbox = document.querySelectorAll('input[type="checkbox"]');
+  chbox.forEach((check, i) => {
+    check.addEventListener('click', () => {
+      todoArray[i].completed = !todoArray[i].completed;
+    });
+  });
+  localStorage.setItem('todoArray', JSON.stringify(todoArray));
+};
```

Comment:
- [ ] The `toggleCheckbox` function adds the click event to each checkbox but it does not update the local storage when the user clicks, only when the `app.js` script run. The project required that any changes made to the list should be preserved, and if the user only updates the checkbox of an item and reload the page, that item's completed status is not being updated. Kindly fix this bug to meet the requirements.